lib/std/json.zig

```diff
@@ -1090,7 +1088,8 @@ pub const TokenStream = struct {
     i: usize,
     slice: []const u8,
     parser: StreamingParser,
-    token: ?Token,
+    token1: ?Token,
+    token2: ?Token,
```

Comment:
I'd consider more descriptive names, e.g. `current_token`, `next_token`

Reply:
I've attempted to maintain the same naming convention the [StreamingParser.feed](https://github.com/ziglang/zig/blob/a5b34a61ab61882bf55d87e4cbc8186215ecf320/lib/std/json.zig#L315-L321) used, but I can change both of them if you think that's better. What do you think?

Reply:
I've thought about it a little bit and I'll just assume you do want this change (to reduce the back-and-forth overhead). I personally think it's better as well

Reply:
I mean, I am absolutely no authority here. Just a random code review from my side since I am interested in having getty/json improved. I suggest finding an author of the code (git blame?) and ping him here to notice the PR. LGTM